frontend/Makefile

```diff
@@ -1,15 +1,21 @@
 SHELL = /bin/bash
 TAG ?= $(shell git describe --exact-match 2>/dev/null)
 COMMIT = $(shell git rev-parse --short=7 HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
+ARO_HCP_FRONTEND_IMAGE_BASE = ${RP_IMAGE_ACR}.azurecr.io/aro
 
 ifeq ($(TAG),)
 	VERSION = $(COMMIT)
 else
 	VERSION = $(TAG)
 endif
 
+ARO_HCP_FRONTEND_IMAGE ?= $(ARO_HCP_FRONTEND_IMAGE_BASE):$(VERSION)
+
 frontend:
 	go build -ldflags "-X main.Version=$(VERSION)" -o aro-hcp-frontend .
 
+image-frontend-multistage:
+	docker build --platform=linux/amd64 --network=host --no-cache -f Dockerfile.frontend -t $(ARO_HCP_FRONTEND_IMAGE) --build-arg REGISTRY=$(REGISTRY) .
```

Comment:
What is the value of `REGISTRY`? if I run `make image-frontend-multistage` using the `frontend/Makefile` `REGISTRY` would expand to an empty value. Could we set a default so it is not the responsibility of the user to pass it? or at least a check where `REGISTRY` is mandatory if we don't want a default